aws-dynamodb-point-in-time-recovery-disabled.yaml

```diff
@@ -0,0 +1,37 @@
+rules:
+- id: aws-dynamodb-point-in-time-recovery-disabled
+  patterns:
+  - pattern: |
+      resource "aws_dynamodb_table" $ANYTHING {
+        ...
+      }
+  - pattern-not-inside: |
+      resource "aws_dynamodb_table" $ANYTHING {
+        ...
+        point_in_time_recovery {
+          enabled = true
+        }
+        ...
+      }
+  message: Point-in-time recovery is not enabled for the DynamoDB table. DynamoDB
```

Comment:
```suggestion
  message: >- Point-in-time recovery is not enabled for the DynamoDB table. DynamoDB
```

Reply:
This is really strange: I'm publishing rules to my private registry in the semgrep app, which contain `>-`, but they show up without this in the app.

Reply:
Here's what it looks like in version control:

<img width="812" alt="Screenshot 2023-02-23 at 11 41 46 AM" src="https://user-images.githubusercontent.com/5559014/220972630-29f8b713-7935-4da1-892f-c580fe71fb6e.png">

Reply:
Here's what it looks like published:

<img width="594" alt="Screenshot 2023-02-23 at 11 42 34 AM" src="https://user-images.githubusercontent.com/5559014/220972802-dafe974c-2772-4502-8166-b85f1ef070ce.png">

Reply:
I'll file some tickets for our App team to sort this out!